### PR TITLE
Version 1.1.0 Databricks Add-on for Splunk

### PR DESCRIPTION
**Pull request by KeerthanaK-18**
Databricks Add-on for Splunk V 1.1.0
Test cases for Databricks Add-on for Splunk V 1.1.0

### CODE REVIEW

**Comment by apamulapati** [comment]
Approved

**Review by arunpamulapati — APPROVED** [review]
Please have a unit test in the labs run to see if the coverage is sufficient.

File: app/bin/databricks_const.py [hunk 1]
```diff
@@ -20,6 +24,11 @@
 
 # Command execution configs
 COMMAND_TIMEOUT_IN_SECONDS = 300
-COMMAND_SLEEP_INTERVAL_IN_SECONDS = 10
+COMMAND_SLEEP_INTERVAL_IN_SECONDS = 3
+
+USER_AGENT_CONST = "Databricks-AddOn/1.1.0 Splunk/{}"
```
Comment (nfx) [inline app/bin/databricks_const.py:29]: it's not allowed to change user agent prefix from `Databricks-AddOnFor-Splunk-`, as internal telemetry depends on it. you can leave ` Splunk/{}` after version sequence number. 

and stability-wise, it's probably 0.2.0 and no way 1.1.0

File: app/README/alert_actions.conf.spec [hunk 2]
```diff
@@ -1,8 +1,7 @@
-
-[notebook]
-python.version          = python3
-param._cam              = <json> Active response parameters.
-param.notebook          = <string> Notebook
-param.paramOne      = <string> Field Name for Parameter One
-param.paramTwo      = <string> Field Name for Parameter Two
-
+[launch_notebook]
```
Comment (nfx) [inline app/README/alert_actions.conf.spec:1]: it's not allowed to change command/parameter names, as people depend on it

**Review by nfx — CHANGES_REQUESTED** [review]
**it's impossible to review this PR**, please immediately split it into at least 

1) removal of `app/bin/ta_databricks` files <= it's great that you're removing these!
2) actual functionality increments.

also change the user-agent string to be compliant with previous one.

otherwise i cannot sign-off on this PR.

**Comment by nfx** [comment]
@KeerthanaK-18 please address the comments above, so that we can proceed with reviewing this PR.

File: app/README/alert_actions.conf.spec (hunk 2 above)
Comment (zaferbil) [reply app/README/alert_actions.conf.spec:1]: This is a breaking change but we can notify users in the change log.

**Pull request closed without merging**